js/deer-utils.js

```diff
@@ -488,6 +488,30 @@ export default {
                 console.log(logMe)
             }
         }
+    },
+
+    /**
+     * Since all Linked Data has the right to be described by multiple contexts, DEER has a special
+     * syntax to support this.  The value of an HTML Attribute DEER.CONTEXT ('deer-context' by default) 
+     * can list multiple URLs like in the example below.
+     *
+     * <span deer-context="[http://example.org/context.json][https://otherexample.org/othercontext.json]"></span>
+     *
+     * Process the value and return either a single string (URL), an array of strings (URLs), or an empty string.
+     *
+     * @see https://stackoverflow.com/questions/1454913/regular-expression-to-find-a-string-included-between-two-characters-while-exclud
+     * @param contextStringValue This is the value from the DEER.CONTEXT attribute on an HTML form.  It is a string.
+    **/
+    processContextSyntax: function(contextStringValue){
+        /**
+         * Note the intricacies of this regex, that perhaps could be improved.
+         *
+         * let b = "[URL_]1][URL_2][URL[_3]"
+         * b.match(/(?<=\[)(.*?)(?=\])/g)
+         * ["URL_", "URL_2", "URL[_3"]
+        */
+        let processedValue = contextStringValue.match(/(?<=\[)(.*?)(?=\])/g)
```

Comment:
I would rather handle `null` values here than to assign a default. If the `ELEMENT.getattribute(DEER-CONTEXT)` does not exist, it will be `null` on the way in here. If it is present without a value, it will be an empty `""`. This may make a difference down the line, so I prefer to preserve that state instead of defaulting to `""` automatically. It's not much of a change from what you've got here.

Suggests:

```javascript
let processedValue = (contextStringValue == null) ? contextStringValue.match(/(?<=\[)(.*?)(?=\])/g) || [] : []
return (processedValue.length>1) ? processedValue : processedValue[0] || ""
```

Reply:
Yeah, that's agreeable.  I am happy just so long as we call it out and handle it in the flow of this functionality somewhere.